### PR TITLE
Conversion rate getter

### PR DESCRIPTION
**Pull request by dave-mills**
Splits + tidies conversion rate handling for IndicatorValue. This lets us to easily get the conversion rate used for any value, both on the front-end and in the export, and it's also a nice refactor of the IndicatorValue and Unit methods! 

The conversion rate should now correctly appear in the export as the "to_standard" multiplier. It's also available ready for updating the front-end.

### CODE REVIEW

File: app/Http/Controllers/Admin/IndicatorValueCrudController.php [hunk 1]
```diff
@@ -170,7 +170,7 @@ protected function setupCreateOperation()
                 'inline_create' => [ 'entity' => 'indicator' ],
                 'minimum_input_length' => 0,
                 'label' => 'Indicator',
-                'hint' => 'If the indicator is not in the dropdown select <b>+Add</b> to add a new one.',
+                'hint' => 'If the indicator is not in the dropdown select <br>+Add</br> to add a new one.',
```
Comment (ciara-mc) [inline app/Http/Controllers/Admin/IndicatorValueCrudController.php:173]: Was this change intentional?

Comment (dave-mills) [reply app/Http/Controllers/Admin/IndicatorValueCrudController.php:173]: It wasn't! Well spotted :)
